docker/base-image/base-server.Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Download dockerize
+FROM alpine:3.12 AS base-server
+
+RUN apk add --no-cache openssl
```

Comment:
You can remove this line and just add `openssl` to `apk add` command bellow.

Reply:
updated and tested